rasa_nlu/emulators/api.py

```diff
@@ -11,12 +16,12 @@ def __init__(self):
 
     def normalise_response_json(self, data):
         # populate entities dict
-        entities = {entity_type: [] for entity_type in set(map(lambda x: x["entity"], data["entities"]))}
+        entities = {entity_type: [] for entity_type in set([x["entity"] for x in data["entities"]])}
         for entity in data["entities"]:
             entities[entity["entity"]].append(entity["value"])
 
         return {
-            "id": unicode(uuid.uuid1()),
+            "id": str(uuid.uuid1()),
```

Comment:
str is here comes from `from builtins import str` which is a backport of the python 3 str class and subclassing the python2 unicode class. This way its behaving similar as before.